src/display/canvas.js

```diff
@@ -705,8 +705,9 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
   CanvasGraphics.prototype = {
 
-    beginDrawing: function CanvasGraphics_beginDrawing(transform, viewport,
-                                                       transparency) {
+    beginDrawing: function CanvasGraphics_beginDrawing({
+      transform, viewport, transparency, backgroundColor = null,
+    }) {
```

Comment:
This is overly verbose, taking full advantage of ES6 notation this can be simplified to:
```javascript
beginDrawing({ transform, viewport, transparency,
               backgroundColor = null, }) {
```
Refer to https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015.